Code/Framework/AzCore/AzCore/Memory/MemoryComponent.h

```diff
@@ -25,32 +25,19 @@ namespace AZ
     public:
         AZ_COMPONENT(AZ::MemoryComponent, "{6F450DDA-6F4D-40fd-A93B-E5CCCDBC72AB}")
 
-        MemoryComponent();
-        virtual ~MemoryComponent();
-
```

Comment:
So what does this component actually do now?
It seems like it can be removed from the AzCoreModules

Reply:
Indeed. I'll go ahead and remove it.

Reply:
Removed in the latest commit.